src/Entity/SnapshotManager.php

```diff
@@ -289,6 +289,30 @@ public function cleanup(PageInterface $page, $keep)
             ));
         }
 
+        if ('mssql' === $platform) {
```

Comment:
This method is already very long... Can you try to extra this to an own method / class